backend/src/Edna/Web/API.hs

```diff
@@ -37,10 +37,7 @@ data EdnaEndpoints route = EdnaEndpoints
   } deriving stock (Generic)
 
 -- | API type specification.
-type EdnaAPI =
-  ToServant EdnaEndpoints AsApi
-  :<|>
-  "health" :> Summary "Check the health of this server" :> GetNoContent
+type EdnaAPI = ToServant EdnaEndpoints AsApi
```

Comment:
Removal of `/health` should be extracted into a separate commit (ideally) or at least be mentioned in the main commit message if you squash everything into one commit (explaining that we can use `/metrics` instead).

Reply:
I'll edit main commit message before merge

Reply:
done